### PR TITLE
Remove prismicData base

### PR DESCRIPTION
**Pull request by Heyrows**
## Description

Remove totally the use & set of .prismic base.

### CODE REVIEW

File: packages/slice-machine/tests/server/env.test.js [hunk 1]
```diff
@@ -211,59 +211,4 @@ describe("getEnv", () => {
     const { env } = await getEnv(TMP);
     expect(env.framework).toEqual("vanillajs");
   });
-
-  test("it should not consider the user logged in if the base from .prismic is different than the one in SM.json", async () => {
```
Comment (MarcMcIntosh) [inline packages/slice-machine/tests/server/env.test.js:215]: probably worth keeping one of these to make sure the cookie is loaded (not sure if that's covered else where)

File: packages/slice-machine/tests/server/services/getPrismicData.test.ts [hunk 2]
```diff
@@ -23,42 +22,21 @@ describe("getPrismicData", () => {
       os.homedir()
     );
 
-    const result = await getPrismicData(base);
+    const result = getPrismicData();
     expect(result.isOk()).toBeTruthy();
-    expect(result.isOk() && result.value.base).toEqual(base);
```
Comment (MarcMcIntosh) [inline packages/slice-machine/tests/server/services/getPrismicData.test.ts:28]: what's base going to be now?

**Review by MarcMcIntosh — COMMENTED** [review]
Nice to see a lot of removals :)

File: packages/slice-machine/tests/server/services/getPrismicData.test.ts (hunk 2 above)
Comment (Heyrows) [reply packages/slice-machine/tests/server/services/getPrismicData.test.ts:28]: getPrismicData doesn't return a base anymore.

Comment (Heyrows) [reply packages/slice-machine/tests/server/services/getPrismicData.test.ts:28]: So the base has to be retrieved from the SM.json like you did

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**